src/api/dataSourceDataGateService.ts

```diff
@@ -76,3 +82,14 @@ export async function getDatasourceDownloadLink(id: string): Promise<string> {
     })
     .then(res => res.url);
 }
+
+export async function createDataSource(name: string) {
```

Comment:
there was no reasons for this function right now because we have no other datasources except File
and probably Datasource - is like abstract class, you can't create just "Datasource", you need to know what are you going to create exactly